ops/DummyMapping/mapping.ts

```diff
@@ -0,0 +1 @@
+import 'allocator/arena';
```

Comment:
What's this?

Reply:
Also as far as I remember this statement became unnecessary a few versions ago...